src/shared/components/date-bubble/index.js

```diff
@@ -1,27 +1,27 @@
 // Displays date bubble
+/* eslint-disable max-len */
 
 import React, { Component } from 'react';
 import styles from './style.css';
 
 export default class DateBubble extends Component {
   static propTypes = {
-    date: React.PropTypes.string,
-    month: React.PropTypes.string,
-    year: React.PropTypes.string,
+    startDate: React.PropTypes.string,
```

Comment:
Sounds like a good idea IMO. `Y M D` as we here lacks time and timezone, and we're just going to put to together to form a Date object anyway.